tools/firtool/firtool.cpp

```diff
@@ -238,11 +239,15 @@ static bool checkBytecodeOutputToConsole(raw_ostream &os) {
 
 /// Print the operation to the specified stream, emitting bytecode when
 /// requested and politely avoiding dumping to terminal unless forced.
-static LogicalResult printOp(Operation *op, raw_ostream &os) {
+static LogicalResult printOp(Operation *op, raw_ostream &os,
+                             bool printLocs = false) {
   if (emitBytecode && (force || !checkBytecodeOutputToConsole(os)))
     return writeBytecodeToFile(op, os,
                                mlir::BytecodeWriterConfig(getCirctVersion()));
-  op->print(os);
+  if (printLocs)
+    op->print(os, OpPrintingFlags().enableDebugInfo());
```

Comment:
Please make this change separately? Emitting debug information in our IR is already controlled by a flag, but regardless we could (should?) have a flag for this in firtool or something? Anyway.